docs/source/routing/get-started.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Apollo Router Quickstart
+subtitle: Run a router binary
+description: This quickstart tutorial walks you through installing an Apollo Router binary, running it with an example supergraph schema and a YAML configuration file, and making test queries with Apollo Sandbox.
+---
+
+import ElasticNotice from "../../shared/elastic-notice.mdx";
+
+Hello! Let's run Apollo Router for the first time, using the simple scenario of developing and deploying locally.
+
+In this guide, you will:
+
+- Download and install a router binary.
```

Comment:
```suggestion
- Download and run the router as a binary.
```
I wouldn't use the word "install" here as they're not installing anything. You're just downloading the router as a binary and running it locally. Would replace "install" with "run" everywhere here